main/acle.rst

```diff
@@ -4508,1153 +3846,1057 @@ following the instruction.
+| UXTAB16         |           | 6, 7E-M       | `__uxtab16`                                                  |
+| UXTAH           |           | 6, 7E-M       | `(uint16_t)x + i`                                             |
+| UXTB16          |           | 6, 7E-M       | `__uxtb16`                                                   |
+| UXTH            |           | 8,6, 6-M      | `(uint16_t)x`                                                 |
+| VFMA            |           | VFPv4         | `fma`, `__fma`                  |
+| VSQRT           |           | VFP           | `sqrt`, `__sqrt`                |
+| WFE             |           | 8,6K, 6-M     | `__wfe`                                                      |
+| WFI             |           | 8,6K, 6-M     | `__wfi`                                                      |
+| YIELD           |           | 8,6K, 6-M     | `__yield`                                                    |
+
+# Advanced SIMD (Neon) intrinsics
+
+## Introduction
 
 The Advanced SIMD instructions provide packed Single Instruction
-Multiple Data (SIMD) and single-element scalar operations on a range
-of integer and floating-point types.
+Multiple Data (SIMD) and single-element scalar operations on a range of
+integer and floating-point types.
 
 Neon is an implementation of the Advanced SIMD instructions which is
 provided as an extension for some Cortex-A Series processors. Where this
-document refers to Neon instructions, such instructions refer to the Advanced
-SIMD instructions as described by the Arm Architecture Reference Manual
-[ARMARMv8]_.
+document refers to Neon instructions, such instructions refer to the
+Advanced SIMD instructions as described by the Arm Architecture
+Reference Manual [[ARMARMv8]](#ARMARMv8).
 
-The Advanced SIMD extension provides for arithmetic, logical and saturated
-arithmetic operations on 8-bit, 16-bit and 32-bit integers (and sometimes
-on 64-bit integers) and on 32-bit and 64-bit floating-point data, arranged
-in 64-bit and 128-bit vectors.
+The Advanced SIMD extension provides for arithmetic, logical and
+saturated arithmetic operations on 8-bit, 16-bit and 32-bit integers
+(and sometimes on 64-bit integers) and on 32-bit and 64-bit
+floating-point data, arranged in 64-bit and 128-bit vectors.
 
 The intrinsics in this section provide C and C++ programmers with a
 simple programming model allowing easy access to code-generation of the
-Advanced SIMD instructions for both AArch64 and AArch32 execution states.
-
-.. _ssec-NEON-intrinsics-concepts:
-
-Concepts
---------
+Advanced SIMD instructions for both AArch64 and AArch32 execution
+states.
 
-The Advanced SIMD instructions are designed to improve the performance of
-multimedia and signal processing algorithms by operating on 64-bit or 128-bit
-*vectors* of *elements* of the same *scalar* data type.
+### Neon Intrinsics Concepts
```

Comment:
Why do we have a title change here? Duplicate targets?

Reply:
Yes

Reply:
ACK, thanks